x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/links/link_to_uptime.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCard } from '@elastic/eui';
+import { I18nProvider } from '@kbn/i18n-react';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { decorateWithGlobalStorybookThemeProviders } from '../../../../../../test_utils/use_global_storybook_theme';
+import { LinkToUptime, LinkTUptimeProps } from './link_to_uptime';
+
+const mockServices = {
+  share: {
+    url: {
+      locators: {
+        get: () => ({
+          navigate: () =>
+            `https://kibana:8080/base-path/app/uptime/?search=host.name: "host1" OR host.ip: "192.168.0.1" OR monitor.ip: "192.168.0.1"`,
+        }),
+      },
+    },
+  },
+};
+
+export default {
+  title: 'infra/Host Details View/Links',
```

Comment:
Just a suggestion: Would it be better to put these stories under `infra/Host Details View/Flyout/...`? 

<img width="230" alt="image" src="https://github.com/elastic/kibana/assets/2767137/806c17bd-d56e-4818-b070-3c36e895f44b">

or even just `Flyout/...` (But maybe this will be better after the flyout becomes an embeddable?

<img width="228" alt="image" src="https://github.com/elastic/kibana/assets/2767137/03936637-7d88-4815-8a1a-4e45bee8b70d">

Reply:
So I will now change it a bit now - as @smith suggested I will add a separate flyout view (living in the hosts view). They are separate from the flyout on purpose because the idea is that only the content of the flyout will be embeddable (not the whole flyout) so I wanted to have separate stories for separate flyout components (which can be used outside of the flyout once they are moved).
The other idea that will be implemented is to have content that can be configurable ( defining which tabs and links should be displayed as a consumer - for example, show only the processes inside an extendable table, show only metadata table on a separate page without filters, etc. ) that way we can show on a story how a certain component will look when it's embedded. I guess that was something I needed to add in the PR description 🙈